openlibrary/utils/proxied_requests.py

```diff
@@ -0,0 +1,12 @@
+import requests
+
+from infogami import config
+
+
+def proxied_request(method, url, **kwargs):
```

Comment:
This approach loses type info when this method is called, making it easy to have typos in parameters/etc. If we can either:

1) Use global env variables; @scottbarnes found that python requests these parameters: https://stackoverflow.com/a/60352993

2) have a helper method, `get_proxies()` so that we can call this like: `requests.get('foo.com', proxies=get_proxies())`. That'll let us maintain the types/auto-completes, and let us use the requests module directly.

Reply:
Update: Actually seeing your other comment about amazon SDKs, I think we might want to go the environment variables route.